app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Filter applications by subject
+description: Let users filter the applications list by the course’s subject
+date: 2021-02-25
+---
+
+We introduced a subject filter to allow users to filter by the course’s subject in the applications list.
+
+## Why we are doing this
+
+Providers need to be able to filter their applications by subject.
+
+When providers share applications with people who do the shortlisting (for example, subject leads in a school or a subject tutor at an HEI), those users only need to see specific subjects and not the entire list.
+
+## An analysis of the course data
+
+From the course data for ‘synced’ providers, there are approximately 40 subjects across primary and secondary levels.
```

Comment:
Instead of quoting synced, can you explain what you mean.

Reply:
Changed to "the providers using the Manage service"